resources/create-sa.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+
+if [ -z "$1" ]; then
+    echo "usage create-sa.sh servicename [ro|rw]"
+    exit 1
+fi
+
+SERVICE="$1"
+
+if [ -z "$2" ] || [[ "$2" == "ro" ]]; then
+    PERMISSION="repository:read,pull:*"
+elif [[ "$2" == "rw" ]]; then
+    PERMISSION="*"
+else
+    echo "usage create-sa.sh servicename [ro|rw]"
+    exit 1
+fi
+
+# create random username suffix and password
+ID=$(doguctl random -l 6 | tr '[:upper:]' '[:lower:]')
+USER="${SERVICE}_${ID}"
+PASSWORD=$(doguctl random)
+
+# connection token
+API_TOKEN=$(doguctl config --encrypted ${CES_TOKEN_CONFIGURATION_KEY})
+
+# create user
+STATUSCODE=$(curl --silent http://localhost:8080/scm/api/v2/users -H "${CES_TOKEN_HEADER}: ${API_TOKEN}" --data '{"name":"'${USER}'","displayName":"CES Serviceaccount","active":true,"password":"'${PASSWORD}'"}' -H "Content-Type: application/vnd.scmm-user+json;v=2" --write-out "%{http_code}")
```

Comment:
I think it would be useful to add the name of the service to the display name. This should make it easier, if we have multiple service accounts.